usaspending_api/common/filters/time_period/time_period.py

```diff
@@ -50,11 +51,31 @@ def gte_date_type(self):
         pass
 
     @abstractmethod
-    def lte_date_type(self):
+    def lte_date_type(self) -> str:
         """
         Returns:
             A date type that indicates the name of a column in which
-            stores the date values that end date must be less than.
+            stores the date values that end date must be less than or equal to.
             Can return None when lte_date_type doesn't exist.
         """
         pass
+
+    @abstractmethod
+    def gte_date_range(self) -> List[dict]:
```

Comment:
Get deeper with the type hints here to really hammer home what's expected:

e.g. is it `List[Dict[str, datetime]` or `List[Dict[str, str]`? 
If the latter, see above thoughts around storing string-formatted dates vs. actual date objects. 

Same with `lte_date_range`

Reply:
Got it. Will do.